apps/web/src/actions/evaluations/create.ts

```diff
@@ -34,13 +27,12 @@ export const createEvaluationAction = authProcedure
     { type: 'json' },
   )
   .handler(async ({ input, ctx }) => {
-    const result = await createEvaluation({
+    const result = await createAdvancedEvaluation({
```

Comment:
what's an advanced evaluation

Reply:
An evaluation where the user defines the prompt (the ones we have now)